students/jemielity_kamil/lesson_05_lists/character_picture_grid.py

```diff
@@ -0,0 +1,14 @@
+grid = [['.', '.', '.', '.', '.', '.'],
+        ['.', 'O', 'O', '.', '.', '.'],
+        ['O', 'O', 'O', 'O', '.', '.'],
+        ['O', 'O', 'O', 'O', 'O', '.'],
+        ['.', 'O', 'O', 'O', 'O', 'O'],
+        ['O', 'O', 'O', 'O', 'O', '.'],
+        ['O', 'O', 'O', 'O', '.', '.'],
+        ['.', 'O', 'O', '.', '.', '.'],
+        ['.', '.', '.', '.', '.', '.']]
+
+for first_loop in range(0, len(grid[0])):
+    for second_loop in range(0, len(grid)):
```

Comment:
Please use starting argument in range method only if different than 0
range(len(grid[0]))